libs/api/mocks/src/domains/license-service/mocks/firearmMock.ts

```diff
@@ -10,46 +10,46 @@ const firearmPropertyDataField = () =>
       {
         type: 'Value',
         label: 'Staða skotvopns',
-        value: faker.random.word(),
+        value: faker.word.sample(),
       },
       {
         type: 'Value',
         label: 'Tegund',
-        value: faker.random.word(),
+        value: faker.word.sample(),
       },
       {
         type: 'Value',
         label: 'Heiti',
-        value: faker.random.word(),
+        value: faker.word.sample(),
       },
       {
         type: 'Value',
         label: 'Númer',
-        value: faker.datatype.number(1000000000).toString(),
+        value: faker.number.int(1000000000).toString(),
       },
       {
         type: 'Value',
         label: 'Landsnúmer',
-        value: faker.datatype.number(99999999).toString(),
+        value: faker.number.int(99999999).toString(),
       },
       {
         type: 'Value',
         label: 'Hlaupvídd',
-        value: faker.datatype.number(50).toString(),
+        value: faker.number.int(50).toString(),
       },
       {
         type: 'Value',
         label: 'Takmarkanir',
-        value: faker.random.word(),
+        value: faker.word.sample(),
       },
     ],
   })
 
 const firearmRightsDataField = () =>
   factory<GenericLicenseDataField>({
     type: 'Category',
-    name: faker.random.alpha({ count: 1, upcase: true }),
-    label: faker.random.word(),
+    name: faker.string.alpha({ length: 1, casing: 'upper' }),
+    label: faker.word.sample(),
```

Comment:
The suggestion looks good by itself, but IMO the fix would be extra work during review as the diff would no longer be technically equivalent. It might be a good improvement that could be tracked/fixed via separate issue/PR after this PR is merged.